geoportailv3/static/js/offline/Configuration.js

```diff
@@ -12,20 +12,32 @@ exports = class extends NgeoConfiguration {
   /**
    * @ngInject
    * @param {!angular.Scope} $rootScope The rootScope provider.
+   * @param {angular.$injector} $injector Main injector.
    * @param {ngeo.map.BackgroundLayerMgr} ngeoBackgroundLayerMgr Background layer
    *     manager.
    */
-  constructor($rootScope, ngeoBackgroundLayerMgr) {
+  constructor($rootScope, $injector, ngeoBackgroundLayerMgr) {
     super($rootScope, ngeoBackgroundLayerMgr);
 
     /**
      * @type {ngeo.map.BackgroundLayerMgr}
      * @private
      */
     this.backgroundLayerMgr_ = ngeoBackgroundLayerMgr;
+
+    let appMymaps = null;
+    if ($injector.has('appMymaps')) {
```

Comment:
Is it necessary to inject `appMymaps` this way?

Reply:
I've some errors (`"app" never provided`) if I inject `app.mymaps` directly. Maybe because we use `goog.module` in this file and `goog.provide` in mymaps ?

Reply:
Yes, maybe. OK for the workaround.